tools/crm_mon.c

```diff
@@ -1132,7 +1132,7 @@ detect_user_input(GIOChannel *channel, GIOCondition condition, gpointer user_dat
             case 'o':
                 show ^= show_operations;
                 if (!pcmk_is_set(show, show_operations)) {
-                    options.mon_ops &= ~mon_op_print_timing;
+                    show ^= show_timings;
```

Comment:
Not the same ... the original cleared mon_op_print_timing, this toggles it. 'o' is meant to toggle operations (only), but this part ensures turning off operations also turns off timings

Reply:
Fixed.